src/Be.Vlaanderen.Basisregisters.MessageHandling.RabbitMq/Be.Vlaanderen.Basisregisters.MessageHandling.RabbitMq.csproj.DotSettings

```diff
@@ -0,0 +1,4 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```

Comment:
We typically use `DotSettings` on a solution level.
For example: https://github.com/Informatievlaanderen/grar-common/blob/main/Be.Vlaanderen.Basisregisters.GrAr.sln.DotSettings

Reply:
Do you want me to copy paste the settings from grar common?

Reply:
yes, that would be great.